projects/bignum-fuzzer/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN apt-get update && apt-get install -y \
 
 # Install Python packages from PyPI
 RUN pip3 install jinja2
+RUN pip3 install jsonschema
```

Comment:
FYI the official Python package requirements are described in [`scripts/basic.requirements.txt`](https://github.com/Mbed-TLS/mbedtls/blob/development/scripts/basic.requirements.txt) (and included files) in the Mbed TLS tree. But I guess that's not practical here since you don't want to have an mbedtls checkout at that point.

Is there a sensible way we could make this information easier to consume, without making it too much of a pain to maintain? (Asking as someone who's not really familiar with Python package management.)

Reply:
I've updated the PR to install the requirements from `basic.requirements.txt`. This should avoid any future build fixes as long as you keep that file updated.

Reply:
Currently, our own CI only tests with all the packages we'll need (`ci.requirements.txt`). We don't yet have an out-of-box test with just `basic.requirements.txt`, but we'll add one at some point.